libraries/Microsoft.Bot.Builder/BotFrameworkAdapter.cs

```diff
@@ -416,6 +416,8 @@ public override async Task<ResourceResponse[]> SendActivitiesAsync(ITurnContext
                 {
                     if (CanProcessOutgoingActivity(activity))
                     {
+                        // If this is a streaming request we need to explicitly fetch a token to make sure it doesn't expire.
```

Comment:
This should be in BotFrameworkHTtpAdapterBase

Reply:
We talked and decided to leave the token fetch here, but add better comments explaining the purpose.